grafana-plugin/src/components/ExtensionLinkMenu/ExtensionLinkMenu.tsx

```diff
@@ -69,7 +67,7 @@ function DeclareIncidentMenuItem({ extensions, declareIncidentLink, grafanaIncid
             category: 'Incident',
             title: 'Declare incident',
             pluginId: 'grafana-oncall-app',
-          },
+          } as any,
```

Comment:
is there an alternative to using `any` here?

Reply:
done